### PR TITLE
DMCA policy updates

### PR DESCRIPTION
**Pull request by vollmera**
We're opening this PR to incorporate our [review process](https://github.blog/2020-11-16-standing-up-for-developers-youtube-dl-is-back/#what-were-changing) for takedown notices alleging violation of the DMCA related to circumvention technology into our DMCA takedown policies. Note, these changes are not new in practice -- we've been applying this process since [announced](https://github.blog/2020-11-16-standing-up-for-developers-youtube-dl-is-back/).

In addition, we are adding an Acceptable Use Policies (AUP) restriction against sharing unauthorized product licensing keys, software for generating unauthorized product licensing keys, or software for bypassing checks for product licensing keys. Related to this, in our DMCA Takedown Policy, we
- explain that the circumvention technology review process does not apply to content that would otherwise violate our AUP because these types of claims are typically straightforward and do not warrant additional technical and legal review process 
- note that where a claim is not straightforward, for example in the case of jailbreaks, the circumvention technology claim review process would apply.

We'll keep this PR open for 30 days and welcome your feedback. Updates to these policies will go into effect after the 30-day notice and comment period, on May 26, at 5pm US Pacific Time.

### CODE REVIEW

**Review by KenG-pro — APPROVED** [review]

File: Policies/dmca-takedown-policy.md
Comment (KOLANICH) [inline Policies/dmca-takedown-policy.md:88]: This sentence is unclear and needs reformulation.

**Comment by sickcodes** [comment]
This is **extremely** broad and should be policed by the vendor the not @github.

**Do not merge.**

The term "jailbreaking" is usually associated with Apple products only and is extremely important for both keeping Apple products secure and the end-users. Most vulnerabilities in Apple are found using jailbroken products.

One such perfect example is the OpenCore bootloader: https://dortania.github.io/OpenCore-Install-Guide/

As a strict good-faith security researcher, who runs a powerful serial number generator for macOS related products that enables researchers to discover flaws in iMessage & Facetime, two critical communication protocols, https://github.com/sickcodes/osx-serial-generator, I have done extensive work in providing access to macOS virtual systems for independent security researchers, and those who can't afford a real mac, but still want to participate in Apple Bug Bounty ($X00,000 bounties).

Apple has very strict requirements: https://developer.apple.com/security-bounty/requirements/ that state, 

You are entirely permitted to violate the EULA of Apple products, if you find and report bugs directly to Apple.

### In fact, Apple ENCOURAGES you to break the EULA to report bugs to them.

> 8. A participant in the Apple Security Bounty program (“ASB Participant”) will not be deemed to be in breach of applicable Apple license provisions which provide that a user of Apple software may not copy, decompile, reverse engineer, disassemble, attempt to derive the source code of, decrypt, modify, or create derivative works of such Apple software, for in scope actions performed by that ASB Participant where all of the following are met:
> >    The actions were performed during good-faith security research, which was — or was intended to be — responsibly reported to Apple;
> >    The actions were performed strictly during participation in the Apple Security Bounty program; and
> >    Neither the actions nor the ASB Participants have otherwise violated these policies such as violating legal requirements 1, 2, and 3, above.

Therefore, **@GitHub should not be the one to police another company's EULA**. A company's EULA is not the law. Moreover, this has nothing to do with DMCA.

In Apple's case, you are given explicit permission that supersedes all other EULAs, so as long as you report the bugs to Apple security team.

Another such cases are WordPress plugins which must be released under the GPLv3+ License, and hence, any such serial number generator is absolutely permitted under the license.

If GitHub feels like becoming the serial number police and hiring a bunch of lawyers to go through each EULA instead of letting the vendor make their own calls, this would limit productivity in security research and ultimately roll-back any current progress that has been made in bridging Developers with Security.

@Github should not get involved in policing serial numbers and this should not be merged.

https://sick.codes/is-hackintosh-osx-kvm-or-docker-osx-legal/

File: Policies/github-acceptable-use-policies.md
Comment (sickcodes) [inline Policies/github-acceptable-use-policies.md:36]: I really don't think random number generators should be added to the DMCA policy.

In any case, https://github.com/github/site-policy/pull/400

**Review by sickcodes — CHANGES_REQUESTED** [review]

**Comment by sickcodes** [comment]
This also lacks software provided under different licenses and should not be merged.

For example, WordPress, under GPLv2+ strictly outlines that, "Derivatives of WordPress code inherit the GPL license."

https://wordpress.org/about/license/

GPL software can contain license periods, and license codes, but it doesn't matter because it's GPL and you are permitted to modify the code, or create license keys.

This license key addition is far too broad.

**Comment by KOLANICH** [comment]
In GPL license it is explicitly written that the software licensed under it explicitly doesn't contain anything that can be considered a DRM for litigation purposes.

File: Policies/github-acceptable-use-policies.md
Comment (jessephus) [reply Policies/github-acceptable-use-policies.md:36]: Thanks. They aren't. :+1:

**Comment by vollmera** [comment]
👋 @sickcodes - just to clarify what this PR is saying 😄 

1. We use an extended review process for circumvention claims - this means we bring in extra lawyers, do a technical review, etc.
2. We are explicitly saying that jailbreaks _would get_ extended review.
3. For this new AUP restriction, we will be using our standard DMCA process (which means the repo owner(s) would be able to dispute the takedown, we'd post the notice in our DMCA repo, etc.) but we will not put those through the extended review for circumvention claims. We are doing this because the review for these categories tends to be straightforward, so there's no need to bring in extra lawyers and do a technical review --- unless it's a potential jailbreak, in which case it would get extended review (see point 2 ☝️ ). The AUP restriction is not intended to change what is a violation of our site policies in practice. 

>@github should not be the one to police another company's EULA.

💯  - GitHub does not police other company's EULAs. If we were to take action on content under our DMCA policy or this AUP restriction, it would be because it's a violation and was reported by the copyright holder.  If someone has permission to post license keys for security research purposes, then it seems unlikely that the copyright holder be reporting them. Even if the copyright holder did report them, they'd be able to push back and show they have permission.

**Review by nkr4449 — APPROVED** [review]

File: Policies/dmca-takedown-policy.md
Comment (vollmera) [reply Policies/dmca-takedown-policy.md]: 👍  I pushed https://github.com/github/site-policy/pull/395/commits/24804cd6b9068c7a41a78248edd70416bffb331a

**Review by KenG-pro — APPROVED** [review]

**Review by pcihon — APPROVED** [review]

**Comment by vollmera** [comment]
Thanks for the feedback! The comment period has ended and the changes are now live 🎉

**Pull request merged**
